.circleci/config.yml

```diff
@@ -95,21 +100,32 @@ jobs:
             ansible-galaxy install -r ansible/requirements.yml
       - run:
           name: Test ansible setup playbook
+          environment:
+            - ANSIBLE_CALLBACK_WHITELIST: profile_tasks
```

Comment:
I see this setting enables logging of timing of Ansible tasks. I see already present at ansible/ansible.cfg though, as these test calls do not `cd` to `ansible/` before running, they do not pick that up. Ok.